Rpackage/R/fitAndScore.R

```diff
@@ -0,0 +1,222 @@
+fit_and_score <- structure(function(target.mat, feature.mat, 
+                                    parameters, n_folds = 3, scorer = NULL, 
+                                    learner = NULL, predict = NULL,
```

Comment:
Predict wil be removed and inferred from learner string

Reply:
done